src/commands/goto.ts

```diff
@@ -108,12 +112,12 @@ function executeGotoLine(editor: vscode.TextEditor, count: number, extend: boole
     }
   }
   
-  editor.selections = editor.selections.map(x =>
-    {
+  editor.selections = editor.selections.map(x => {
       const npos:vscode.Position = new vscode.Position(x.active.line,  (positions[position] || positions['default'])(x));
       return new vscode.Selection(extend ? x.anchor : npos, npos)
     }
   )
+  return Promise.resolve()
 }
 
 function executeGotoDisplayLine(editor: vscode.TextEditor, count: number, extend: boolean, position: string) {
```

Comment:
Same thing here: setting `position`'s type to `'top' | 'center' | 'bottom' | 'default'` will improve type-checking (and we should remove the annotation on `positions` to further improve type-checking).